docs/zq/functions/unflatten.md

```diff
@@ -1,23 +1,24 @@
 ### Function
 
-&emsp; **unflatten** &mdash; transform a record with dotted names to a nested record
+&emsp; **unflatten** &mdash; transform an array of key/value records into a
+record.
 
 ### Synopsis
 
 ```
-unflatten(val: record) -> record
+unflatten(val: [{key:[string],value:any}) -> record
 ```
 ### Description
-The _unflatten_ function returns a copy of `val` with all dotted field names converted
-into nested records. If no argument is supplied to `unflatten`, `unflatten`
-operates on `this`.
+The _unflatten_ functions converts the key/values records in array `val` into
+a single record. _unflatten_ is the inverse of _flatten_ i.e., `unflatten(flatten(r))`
+will produce an equivalent record to `r`.
 
 ### Examples
 
```

Comment:
Maybe lead with an example of `unflatten` on its own, like `echo '[{key:"a",value:1},{key:"b",value:2}] | zq 'yield unflatten()' -`?